src/librustc_target/spec/thumbv4t_nintendo_gba.rs

```diff
@@ -0,0 +1,80 @@
+//! Targets the Nintendo Game Boy Advance (GBA),
+//! a handheld game device from 2001.
+//!
+//! Please ping @Lokathor if changes are needed.
+//!
+//! The target profile assumes that you have the ARM binutils in your path (specifically the linker, `arm-none-eabi-ld`). They can be obtained for free for all major OSes from the ARM developer's website, and they may also be available in your system's package manager
```

Comment:
Can we use rust-lld perhaps? That would make this work out of the box which would be great.

Reply:
quite probably, that's also being discussed [in another review comment](https://github.com/rust-lang/rust/pull/74419#discussion_r456139623). I'm going to mark this comment as resolved and we can focus the discussion there.